### PR TITLE
[WIP] Mark vibe.crypto.passwordhash as deprecated

### PR DESCRIPTION
**Pull request by kubo39**
~~Remove deprecated passwordhash module, it's insecure.~~
vibe.crypto.passwordhash is not secure, so mark this as deprecated.
And all functions in this module were already marked as deprecated, so just remove.

### CODE REVIEW

**Comment by Geod24** [comment]
Hum, why was the module not deprecated ? I agree with removing the functions, but it could break packages with lingering imports.

**Comment by kubo39** [comment]
> why was the module not deprecated?

I'm not sure, I saw [this issue](https://github.com/vibe-d/vibe.d/issues/794) but didn't mention for it.

Would it be better to mark the passwordhash module as deprecated and remove all functions in this module?

**Comment by Geod24** [comment]
> Would it be better to mark the passwordhash module as deprecated and remove all functions in this module?

Yes

File: crypto/vibe/crypto/passwordhash.d [hunk 1]
```diff
@@ -5,89 +5,5 @@
 	License: Subject to the terms of the MIT license, as written in the included LICENSE.txt file.
 	Authors: Sönke Ludwig
 */
+deprecated("This module will be removed(see #794). Phobos's std.digest.hmac or the DUB packages dauth or scrypt may be suitable alternatives.")
```
Comment (kubo39) [inline crypto/vibe/crypto/passwordhash.d]: I just added std.digest.hmac module. Is there any suggestion for?

Comment (kubo39) [reply crypto/vibe/crypto/passwordhash.d]: I was wrong, hmac is not suitable for.

**Pull request merged**